### ISSUE TITLE
Milestone - Harmonizing environmental science standards support linking of climate data (2.9)

### ISSUE THREAD

**Issue by ssarrafan** [microbiomedata/issues#502]
Environmental science standards to support linking with microbiome data
While there are many existing challenges to fully leverage microbiome data for climate science (11), a
tangible, near-term challenge that can be addressed is the need to support linked data standards for
relevant environmental processes, environmental variables, and environment types. Similar challenges
have been addressed for genomic medicine applications where gene and health outcome data standards
were modeled (e.g., Fast Healthcare Interoperability Resources (15) and Observational Medical Outcomes
Partnership (16)) and standardized vocabularies such as Systematized Nomenclature of Medical Terms
(17)) were developed. To study the association between microscale processes all the way to the
ecosystem scale, it is essential to compare variables across studies through meta-analyses which is only
possible through standardizing data. Thus, we aim to catalyze interoperable ‘planetary health records’
through the creation of metadata crosswalks between all elements in the NMDC schema and multiple
different environmental standards (Milestone 2.9).

Page 32

**Comment by ssarrafan** [comment]
@cmungall this is due this quarter. Any update?  Will this be done by September?

**Comment by ssarrafan** [comment]
Discussed with Emiley and Alicia today.  Emiley would like to discuss this milestone with @shreddd to determine if we should focus on this milestone.

**Comment by ssarrafan** [comment]
@emileyfadrosh @shreddd did you discuss this milestone? 
@sierra-moxon anything I can add for this DOE report for this milestone?

**Comment by sierra-moxon** [comment]
There have been several pull requests initiated and merged in the environmental ontology repo including term fixes to harmonize with GOLD: [https://github.com/EnvironmentOntology/envo/pull/1517](https://www.google.com/url?q=https://github.com/EnvironmentOntology/envo/pull/1517&sa=D&source=docs&ust=1727712058801551&usg=AOvVaw1UprT7jLb-gEZEfhGoTOSa)
[https://github.com/EnvironmentOntology/envo/pull/1519](https://www.google.com/url?q=https://github.com/EnvironmentOntology/envo/pull/1519&sa=D&source=docs&ust=1727712058801600&usg=AOvVaw0OzJo_LPtpXE03p43vqeS8) - new release of ENVO
ecoregion updates: [https://github.com/EnvironmentOntology/envo/pull/1523](https://www.google.com/url?q=https://github.com/EnvironmentOntology/envo/pull/1523&sa=D&source=docs&ust=1727712058801622&usg=AOvVaw2G5H0W0eC5UICwmzF3nL6P), [https://github.com/EnvironmentOntology/envo/pull/1526](https://www.google.com/url?q=https://github.com/EnvironmentOntology/envo/pull/1526&sa=D&source=docs&ust=1727712058801636&usg=AOvVaw0xMwp1IM51ab-LUDzd6iDv)
animal constructs: [https://github.com/EnvironmentOntology/envo/pull/1530](https://www.google.com/url?q=https://github.com/EnvironmentOntology/envo/pull/1530&sa=D&source=docs&ust=1727712058801651&usg=AOvVaw23TtJcn1E3RzPoXp7EDi32)
bioremediation: [https://github.com/EnvironmentOntology/envo/pull/1534](https://www.google.com/url?q=https://github.com/EnvironmentOntology/envo/pull/1534&sa=D&source=docs&ust=1727712058801666&usg=AOvVaw18RoRzsE4nO3J3FJFM4xI5)
I think these all relate to the larger milestone of harmonizing environmental standards

**Comment by sierra-moxon** [comment]
The work from the ENV triad group has also identified several issues with ENVO broad, medium, and local scale terms, currently documented in a set of summary spreadsheets, mostly centering around identifying and reconciling the axis of differentiation in these terms sets.  We expect ENVO changes to come out of this effort as well as a constrained set of terms for each environment/module.   In addition we are working steadily towards a set of reusable scripts that help us identify and extract ENVO subsets to use in constraining the submission portal and field notes application to help users choose terms.
Finally, we have proposed and are working on a generic ontology loader into NMDC as well as a node normalizer so that user-entry into our system is more robust (they can have access to definitions, sy synonyms, cross references and more with the ENV triad terms. This will help users pick consistent terms to define their samples .  This helps harmonize the samples across our data sets.

**Comment by sierra-moxon** [comment]
I do not believe this is completed, and should be an ongoing milestone (as I understand it)

**Comment by ssarrafan** [comment]
@sierra-moxon @cmungall when do you anticipate completing this milestone so I can move it to the appropriate quarter?

FYI @emileyfadrosh

**Comment by turbomam** [comment]
I'm rcurrently unning a lexical match between the labels, aliases, etc. of nmdc-schema terms (including the MIxS terms) against the labels, aliases etc. of the terms in many databases at  https://s3.amazonaws.com/bbop-sqlite. 

That overlaps a lot with ontologies in BioPortal (which is generally an superset of ontologies in OLS or ontologies in the OBO foundry.)

I can give an update on this and it applicability to this milestone in one of our meetings next week. NMDC sync maybe?

**Comment by turbomam** [comment]
Lexical matching against several of the knowledgebases in https://github.com/INCATools/semantic-sql/blob/main/src/semsql/builder/registry/ontologies.yaml 
 
- https://github.com/microbiomedata/external-metadata-awareness/blob/main/notebooks/multi-lexmatch/lexmatch-combined.tsv 

Doesn’t yet include knowledgebases that don’t have an OWL version in OBO, BioPortal or BBOP 
Like CF [Climate Forecast Conventions](https://cfconventions.org/vocabularies.html)

**Comment by ssarrafan** [comment]
New due date for this issue: 2025Q4
Work to begin this quarter per planning meeting today
FYI @cmungall @mslarae13
